sql/parser/sql.y

```diff
@@ -861,14 +861,14 @@ iso_level:
     // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| REPEATABLE READ
+| SNAPSHOT
   {
-    // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| SNAPSHOT
+| REPEATABLE READ
   {
-    $$ = SnapshotIsolation
+    // Mapped to the closest supported isolation level.
```

Comment:
Looking at this months after I wrote it, I wonder if it would be better to error when the user asks for `REPEATABLE READ`?